package.json

```diff
@@ -53,7 +53,7 @@
     "dependencies": {
         "@grpc/grpc-js": "^1.6.7",
         "@hashgraph/cryptography": "^1.1.2",
-        "@hashgraph/proto": "^2.4.1",
+        "@hashgraph/proto": "link:packages/proto",
```

Comment:
Is proto still behind or can this be updated?

Reply:
This can be updated, and it has been updated in the release branches. Will make sure to update this here as well.